### PR TITLE
Updated Romanian Translation

### PR DESCRIPTION
**Pull request by LuisPavelA**

### CODE REVIEW

**Comment by LuisPavelA** [comment]
Ready to merge? What do I do now? I am new to this.

File: kibbeh/public/locales/ro/translation.json [hunk 1]
```diff
@@ -37,11 +37,11 @@
       "currentRoom": "Deocamdată în:",
       "startPrivateRoom": "Începe o cameră privată cu ei"
     },
-    "followList": { "followHim": "follow", "followingHim": "following" },
+    "followList": { "followHim": "follows", "followingHim": "following" },
```
Comment (theotterlord) [inline kibbeh/public/locales/ro/translation.json]: Is the extra `s` the actual translation?

**Review by TheOtterlord — COMMENTED** [review]
Just a quick question

File: kibbeh/public/locales/ro/translation.json (hunk 1 above)
Comment (LuisPavelA) [reply kibbeh/public/locales/ro/translation.json]: I think that is a mistake

Comment (LuisPavelA) [reply kibbeh/public/locales/ro/translation.json]: how do I change it?

Comment (theotterlord) [reply kibbeh/public/locales/ro/translation.json]: You can change your pull-request by looking at files in your [fork](https://github.com/LuisPavelA/dogehouse/tree/patch-1). The file you are looking for is [here](https://github.com/LuisPavelA/dogehouse/blob/patch-1/kibbeh/public/locales/ro/translation.json). You can edit it in the browser, or if you have cloned the fork to your computer, you can edit it there and push to your fork.

Comment (LuisPavelA) [reply kibbeh/public/locales/ro/translation.json]: I can't find the mistake in my fork :(

Comment (HarrisonMayotte) [reply kibbeh/public/locales/ro/translation.json]: Yeah it appears there isn't a mistake like that on the fork on my end either.

Comment (HarrisonMayotte) [reply kibbeh/public/locales/ro/translation.json]: The most recent commit appears to resolve this.

**Pull request merged**